src/main/target/AG3X/target.c

```diff
@@ -60,4 +60,6 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     // I2C2
     DEF_TIM(TIM2,  CH3, PB10, TIM_USE_NONE,   0, 0), // SCL2   D(1,1)
     DEF_TIM(TIM2,  CH4, PB11, TIM_USE_NONE,   0, 0), // SDA2   D(1,7) D(1,6)
+    BBSHOT_PACER_TIMER_CHANNELS
```

Comment:
Why are these added twice?

Also, instead of modifying all target definitions, this should really be done by making the code consuming `timerHardware` aware of bitbanging Dshot.

Also, as remarked earlier, this approach will not work for Unified Targets, so we need to look for a different approach.

Reply:
Yeah this was addressed with the separate bbTimerHardware array.